src/common/utils/utils.ts

```diff
@@ -164,13 +164,14 @@ const copyToClipboard = (text: string): void => {
   navigator.clipboard.writeText(text);
 };
 
+// TODO: should type properly (especially return type should not be `any`)
 const getRandomVaultIdWithCapacity = (
```

Comment:
@anton-karlovskiy this utility is also being used on the redeem form - have you tested to make sure there's no regression?

Reply:
@tomjeatt 
Yes, I tested but as it's not used in the issue form, I reverted the changes.